src/web/modules/payment_links/overview.njk

```diff
@@ -0,0 +1,66 @@
+{% from "transactions/status.macro.njk" import status %}
+{% from "common/json.njk" import json %}
+{% extends "layout/layout.njk" %}
+
+{% block main %}
+  <span class="govuk-caption-m">
+    {% if account %}
+      <span>{{ account.name }}</span>
+    {% else %}
+      <span>GOV.UK Pay platform</span>
+    {% endif %}
+  </span>
+
+  <h1 class="govuk-heading-m">Payment links</h1>
+
+  {% if filters.reference %}
+  <div class="govuk-body">
+    <span>Filtered by reference</span>
+    <span><strong class="govuk-tag">{{ filters.reference }}</strong></span>
+  </div>
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  {% endif %}
+
+  <div>
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">ID</th>
+          <th class="govuk-table__header" scope="col">Name</th>
+          <th class="govuk-table__header" scope="col">Type</th>
```

Comment:
reflecting on our conversation, we probably don't want a "type", they should all be "ADHOC"